content/en/docs/components/kfserving/webapp.md

```diff
@@ -0,0 +1,295 @@
++++
+title = "Models UI"
+description = "Web app for managing Model servers"
+weight = 2
+
++++
+
+The Models web app is an abstraction that provides users a graphical UI to
+manage their Model servers, by performing CRUD operations on
+top of `InferenceService`
+[CustomResources](https://github.com/kubeflow/kfserving/tree/master/pkg/apis/serving).
+
+The web app is also exposing information from the underlying Knative resources,
+like Conditions from the Knative Configurations, Route and Revisions as well as
+live logs from the Model server pod.
+
+## Installation and Access
+
+The web app's manifests are part of the upstream [KFServing
+manifests](https://github.com/kubeflow/kfserving/tree/master/config/web-app).
+This means that the neccessary resources will be deployed when installing
+KFServing. KFServing manifests for the [0.6 release](https://github.com/kubeflow/kfserving/tree/release-0.6)
+come with two flavors, _standalone_ and _kubeflow_. The web app's
+manifests are part of both installation options.
+
+The web app includes the following resources:
+* A `Deployment` for running the backend server, and serving the static frontend files
+* A `Service` for configuring the incluster network traffic
+* A `ServiceAccount` and `ClusterRole{Binding}` to give the necessary
+    permissions to the web app's Pod
+* A `VirtualService` for exposing the app via the cluster's Istio Ingress
+    Gateway
+
+### Standalone
+In this case all the resources of the web app will be installed in the
+`kfserving-system` namespace. Users can access the web app either via the
+`knative-ingress-gateway.knative-serving` Istio Ingress Gateway or by
+port-forwarding the backend.
+
+#### Port forwarding
+```bash
+# set the following ENV vars in the app's Deployment
+kubectl edit -n kfserving-system deployments.apps kfserving-models-web-app
+# APP_PREFIX: /
+# APP_DISABLE_AUTH: "True"
+# APP_SECURE_COOKIES: "False"
+
+# expose the app under localhost:5000
+kubectl port-forward -n kfserving-system svc/kfserving-models-web-app 5000:80
+```
+
+### Kubeflow
+The web app is not part of the Kubeflow [1.3
+release](https://github.com/kubeflow/manifests/tree/v1.3-branch) manifests. The web
+app was introduced in KFServing `0.6` and the Kubeflow manifests for `1.3`
+include KFServing `0.5`.
+
+The KFServing manifests include a `kubeflow` overlay, which you can apply in
```

Comment:
Maybe we can publish an addon yaml for web app, the KFServing 0.6 release requires a configuration change to work with Kubeflow 1.3 as the Knative local gateway changes.

Reply:
> Maybe we can publish an addon yaml for web app

@yuzisun what would this addon contain? For the web app to work alongside Kubeflow (1.3) it only needs a `VirtualService` that will use the default Istio ingress gateway that all Kubeflow services use to get exposed.

Or did you had something completely different in mind?

Reply:
@kimwnasptd Kubeflow 0.6 changed the default local gateway from cluster local gateway to knative local gateway which requires higher version of knative while Kubeflow by default still installs 0.17,  will need to add a note for the configuration if user applies the 0.6 yaml to their kubeflow 1.3 installation.

Reply:
Added a commit to note that users should edit their `inferenceservice-config` and use the corresponding local gateway if they installed KFServing 0.6 with Kubeflow 1.3